wagtail/admin/static_src/wagtailadmin/scss/layouts/login.scss

```diff
@@ -114,16 +118,25 @@ input[type=checkbox]:before {
     line-height: 1.6em;
 }
 
+// Forgotten pasword link
 .help {
-    opacity: 1;
-    position: absolute;
     font-size: 1.3em;
-    margin-top: -0.5em;
-    right: 5%;
-    top: 213px;
+    margin-top: 0;
+    padding: 10px 0;
 
     @include media-breakpoint-up(sm) {
-        top: 65%;
+        position: absolute;
+        top: 139px;
+        right: 5%;
+        padding: 0;
+    }
+
+    &__link {
+        color: $color-white;
+
+        @include media-breakpoint-up(sm) {
+            color: $color-link;
```

Comment:
Because this resolves to a css custom property, this remains white in IE11 at desktop.

Reply:
I have now added a fallback colour to address this.

Reply:
We officially no longer support IE11 starting with the next release, so I will remove the fallback now.

More generally, the linting rule is there to prevent unintended usage of the same property twice. You’d add a stylelint-disable comment in a case like, mentioning this is intentional / why.